README.md

```diff
@@ -26,30 +26,27 @@ implementations are ill-suited for container environments:
   centralized logging when a syslog server is running, but with containers,
   simply logging to `stdout` or `stderr` is preferred.
 
-Finally, they are often very quiet, which makes the above issues difficult to
-understand or debug!
+Finally, they are often quiet, making these issues difficult to understand and debug!
 
-The list could go on, but the fundamental takeaway is this: unlike typical
-server cron implementations, Supercronic tries very hard to do exactly what you
-expect from running `cron` in a container:
+Supercronic's goal is to behave how you would expect `cron` running in a container to behave:
 
-- Your environment variables are available in jobs.
-- Job output is logged to `stdout` / `stderr`.
+- Your environment variables are available in jobs
+- Job output is logged to `stdout` / `stderr`
 - `SIGTERM` (or `SIGINT`, which you can deliver via CTRL+C when used
   interactively) triggers a graceful shutdown
-- Job return codes and schedules are also logged to `stdout` / `stderr`.
+- Job return codes and schedules are also logged to `stdout` / `stderr`
 
 ## How does it work? ##
 
-- Install Supercronic (see below).
-- Point it at a crontab: `supercronic CRONTAB`.
+- Install Supercronic (see below)
+- Point it at a crontab: `supercronic CRONTAB`
 - You're done!
```

Comment:
This is an exception to the consistency rule. The rest of the listworks better without the periods (especially after the code block), making the exclamation point a nice little surprise at the end.